src/containers/editor/graph/Node.tsx

```diff
@@ -32,7 +40,7 @@ export const ObjectNode = memo(({ id, data }: NodeProps<NodeWithData>) => {
   return (
     <>
       {data.toolbarVisible && <Toolbar id={id} />}
-      <div className="nodrag nopan graph-node cursor-default" style={data.style}>
+      <div className={clsx("nodrag nopan graph-node cursor-default", { "opacity-50": !!unfoldNodeMap[id] })}>
```

Comment:
I think we need to use `node.hidden` of the child instead, as it makes the code more cohesive if we also support opacity on fold siblings. I'm going to remove `unfoldNodeMap` from `statusStore`.

Reply:
The logic is updated to use a function `getNode` with `node.childrenKey2Id`. Can you review the logic? Thank you.

Reply:
Can you review the logic? I updated it. Thank you